dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipService.java

```diff
@@ -74,6 +74,15 @@
      */
     Relationship getRelationship( long id );
 
+    /**
+     * Checks for the existence of a relationship by UID. Takes into account
+     * also the deleted relationships.
+     *
+     * @param uid Relationship UID to check for.
+     * @return relationship exists or not.
```

Comment:
I know the return one in the style of

`return true if relationship exists, false otherwise`